src/controllers/controllerengine.cpp

```diff
@@ -1030,18 +1080,22 @@ bool ControllerEngine::evaluate(QString scriptName, QList<QString> scriptPaths)
     }
     if (error != "") {
         error = QString("%1 at line %2, column %3 in file %4: %5")
-                    .arg(error,
-                         QString::number(result.errorLineNumber()),
-                         QString::number(result.errorColumnNumber()),
-                         filename, result.errorMessage());
+                        .arg(error, QString::number(result.errorLineNumber()),
+                             QString::number(result.errorColumnNumber()),
+                             filename, result.errorMessage());
```

Comment:
Well, here's one case where I prefer the previous organization. :) But this is an exception.